src/index.js

```diff
@@ -1 +1,63 @@
+/* eslint-disable import/no-extraneous-dependencies */
 import './style.css';
+import 'babel-polyfill';
+
+const refresh = document.getElementById('refreshbtn');
+const submit = document.getElementById('submit');
+const user1 = document.getElementById('user');
+const score1 = document.getElementById('score');
+const ul = document.getElementById('list');
+const id = 'myleaderboard1234';
+const fetchURL = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${id}/scores/`;
+
+const postData = async () => {
+  if (user1.value === '' || score1.value === '') {
+    // empty
+  } else {
+    try {
+      await fetch(fetchURL, {
+        method: 'post',
+        body: JSON.stringify({
+          user: user1.value,
+          score: score1.value,
+        }),
+        headers: {
+          'Content-type': 'application/json; charset=UTF-8',
+        },
+      });
+      window.location.reload();
+    } catch (error) {
+      // empty
+    }
+  }
+};
+
+refresh.addEventListener('click', () => {
+  window.location.reload();
+});
```

Comment:
- [ ] It would be so much better if the refresh button hit the API and refresh the score. Currently, what `refresh` is doing is refreshing the page and that is not what should be refresh.  Kindly just add the button `refresh` to the get score same as the `submit button. Don't forget to delete the refresh page action on the postData function. 

😄 🛠️

Reply:
:heavy_check_mark: Well resolved